SIL.Windows.Forms/ClearShare/WinFormsUI/ContributorsListControl.cs

```diff
@@ -56,7 +59,11 @@ private void Initialize()
 			col.Width = 120;
 			_grid.Columns.Add(col);
 
-			_grid.Columns.Add(BetterGrid.CreateCalendarControlColumn("date", "Date", null, CalendarCell.UserAction.CellMouseClick));
+			var dateColumn = BetterGrid.CreateCalendarControlColumn("date", "Date", null,
+				CalendarCell.UserAction.CellMouseClick);
+			_grid.Columns.Add(dateColumn);
+			if (_disableDateColumn)
+				dateColumn.Visible = false;
```

Comment:
I'm unfamiliar with BetterGrid. I'm assuming you're still adding `dateColumn` so the user can show it if desired. In this case, I think a better name for the new constructor argument might be `dateColumnVisible`.

Reply:
It also seems you could do away with the new constructor parameter and accomplish the same goal by modifying `model.ContributorGridSettings`